usvm-core/src/main/kotlin/org/usvm/StepScope.kt

```diff
@@ -180,12 +207,15 @@ class StepScope<T : UState<Type, *, Statement, Context, *, T>, Type, Statement,
             return fork(condition, blockOnTrueState, blockOnFalseState)
         }
 
+        // If condition is concrete there is no fork point possibility
+        val registerForkPoint = !condition.isConcrete
```

Comment:
Why there is no option to disable fork registration at all as it made in other methods?

Reply:
There is no such option for other methods. This should be more clear after renaming